### PR TITLE
Created the first recipe for automan

### PR DESCRIPTION
**Pull request by rishi9699**
Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/recipe/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    # If your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # Compilers are named 'c', 'cxx' and 'fortran'.
```
Comment (scopatz) [inline recipes/recipe/meta.yaml]: please remove comments

File: recipes/recipe/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - {{ compiler('c') }}
```
Comment (scopatz) [inline recipes/recipe/meta.yaml]: if this is noarch python you shouldn't need compilers

File: recipes/recipe/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+about:
+  home: https://github.com/pypr/automan
+  license: MIT
+  license_file: LICENSE.txt
+  summary: 'automan'
+
+
+extra:
+  recipe-maintainers:
+    - prabhuramachandran
```
Comment (scopatz) [inline recipes/recipe/meta.yaml]: @prabhuramachandran Are you OK with being a maintainer here?

Comment (prabhuramachandran) [reply recipes/recipe/meta.yaml]: Yes, that should be OK.  For the time being @rishi9699 is going to setup the recipes and make sure that it works correctly.

**Pull request closed without merging**